src/appkit.cs

```diff
@@ -3891,6 +3973,21 @@ public interface NSColorSpace : NSCoding, NSSecureCoding {
 		[Export ("availableColorSpacesWithModel:")]
 		NSColorSpace [] AvailableColorSpacesWithModel (NSColorSpaceModel model);
 
+		[Mac (10, 12)]
+		[Static]
+		[Export ("extendedSRGBColorSpace")]
+		NSColorSpace ExtendedSRGBColorSpace { get; }
```

Comment:
How about ExtendedSRgbColorSpace? ExtendedsRgbColorSpace looks really weird, like it's Extendeds Rgb